tests/files/BlockchainTests/bcUncleTest.json

```diff
@@ -4826,23 +4826,23 @@
         "lastblockhash" : "eed1b4da708283370856fc76352d68f36d9766b7f366da372e2992ced9a1f663",
         "postState" : {
             "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
-                "balance" : "0x14",
+                "balance" : "0x1e",
```

Comment:
What was changed in this test that the post state is different but none of the inputs or transitions changed?